.drone.yml

```diff
@@ -0,0 +1,206 @@
+---
+kind: pipeline
+type: docker
+name: x86_64-linux
+platform:
+  os: linux
+  arch: amd64
+
+# Some expressions in this file are duplicates. Scripting support is
+# available using jsonnet but increases complexity
+# https://docs.drone.io/pipeline/scripting/jsonnet/
+
+# These volumes will be mounted at the build directory, allowing to
+# run different premake toolchains from the same source tree
+volumes:
+- name: build-premake
+  temp: {}
+- name: build-cmake
+  temp: {}
+
+steps:
+#
+# Setup the source tree
+#
+- name: clone-submodules
+  image: xeniaproject/buildenv:2021-05-05
+  commands:
+  - pwd
+  # May miss recursive submodules (but faster than xb setup)
+  - git submodule update --init --depth 1 -j $(nproc)
+
+
+#
+# Setup the two build systems
+#
+
+# Native premake Makefiles for production
+- name: toolchain-premake
+  image: xeniaproject/buildenv:2021-05-05
+  volumes:
+  - name: build-premake
+    path: /drone/src/build
+  commands:
+  - $CXX --version
+  - $AR --version
+  - python3 --version
+  - ./xenia-build premake
+  depends_on:
+    - clone-submodules
+
+# Development toolchain
+- name: toolchain-cmake
+  image: xeniaproject/buildenv:2021-05-05
+  volumes:
+  - name: build-cmake
+    path: /drone/src/build
+  commands:
+  - |
+    ./xenia-build premake --devenv=cmake
+    cd build
+    for c in Debug Release
+    do
+      mkdir cmake-$c
+      cd cmake-$c
+      cmake -DCMAKE_BUILD_TYPE=$c ..
+      cd ..
```

Comment:
```suggestion
      pushd cmake-$c
      cmake -DCMAKE_BUILD_TYPE=$c ..
      popd
```

Reply:
I prefer it to be explicit here.
Also we only go in one directory.
It would additionally print a line, making this more verbose than required